scripts/buildTypes.js

```diff
@@ -26,6 +26,11 @@ function rewriteImportPath(importPath) {
     return importPath.replace(stylesSrcPath, '@material-ui/styles');
   }
 
+  const systemSrcPath = path.posix.join('..', 'material-ui-system', 'src');
```

Comment:
The build was failing without this, as the system is referenced in the `experimentalStyled()`.

Reply:
We can strip these fixes in TypeScript 4.3. Their [current dev build no longer requires manual rewriting](https://app.circleci.com/pipelines/github/mui-org/material-ui/41571/workflows/ee91f829-23a4-44b2-856e-b51a35f6a239/jobs/248247/parallel-runs/0/steps/0-111).

Reply:
Perfect, I was confused why it was failing at the beginning. It's great that we can drop them 👍